examples/simple-udp/main.go

```diff
@@ -144,7 +144,17 @@ func readWriteLoop(conn net.PacketConn, stop chan struct{}, s *sdk.SDK) {
 			allocate(s)
 
 		case "RESERVE":
-			reserve(s)
+			if len(parts) == 2 {
+				if dur, err := time.ParseDuration(parts[1]); err != nil {
+					respond(conn, sender, err.Error()+"\n")
```

Comment:
Should this response also be prefixed with "ERROR:"?

Reply:
Yes, yes it should! 

(And this is why I ask for review before pushing images :smile:)

Let me know if there is anything else, and I'll push the image so the e2e tests can run.